src/utl/include/utl/deleter.h

```diff
@@ -10,6 +10,6 @@
 namespace utl {
 
 template <typename T>
-using deleted_unique_ptr = std::unique_ptr<T, std::function<void(T*)>>;
+using deleter_unique_ptr = std::unique_ptr<T, std::function<void(T*)>>;
```

Comment:
If we are fixing the name it would also be good to make it adhere to Google style where type names are PascalCase: DeleterUniquePtr.  Personally I would choose UniquePtrWithDeleter even though its a bit more verbose.

Reply:
Done.